### ISSUE TITLE
V6.0.1 Install Failure

### ISSUE THREAD

**Issue by CrackBaby1** [Ic0nic0de/TheCuratorsCompanion#23]
I attempted to update today and received the following error in Vortex.

![TCCInstallError](https://user-images.githubusercontent.com/60897758/123176453-b08da380-d440-11eb-8165-cce0d7c101aa.JPG)

I deleted and re-downloaded the file in case there'd been some corruption in the original download, but this failed to solve the issue.  I also attempted a manual download to see if Vortex was messing up the file, but also to no avail.

**Comment by CrackBaby1** [comment]
Not sure if this helps, but after updating Vortex (which I had stupidly failed to do before), I got the following additional information.

![TCCInstallFail2](https://user-images.githubusercontent.com/60897758/123181006-a9b75e80-d449-11eb-9791-31abe7276050.png)

**Comment by CrackBaby1** [comment]
Sorry to spam this up, I should've done a bit more research before posting the first, much less the second time.  So as an experiment, I modified the ModuleConfig.xml file in the fomod folder, completely removing the last two TCC options (the ones where the above noted errors occur).  This allowed me to install in Vortex without any issues.  Based on how the rest of the file looks for adding options, it appears that it's just missing a couple lines in the last two TCC entries for the required dependencies, which is upsetting Vortex.  Add the dependencies (or remove the entire two blocks of code like I did), and problem vanishes.
